### ISSUE TITLE
"Beginner" text has insufficient color contrast

### ISSUE THREAD

**Issue by schne324** [dequelabs/workshop-day-II#3]
https://axe.deque.com/issues/02b8d307-19f6-4905-94df-56901a90658d
